diff --git a/libfwupdplugin/meson.build b/libfwupdplugin/meson.build
index 71667b6bc..4dc7b9c52 100644
--- a/libfwupdplugin/meson.build
+++ b/libfwupdplugin/meson.build
@@ -300,7 +300,8 @@ if introspection.allowed()
       girtargets,
       fwupd_gir[0],
     ],
-    install: false
+    install: false,
+    install_dir_typelib: libdir_pkg / 'girepository-1.0',
   )
 
   # Verify the map file is correct -- note we can't actually use the generated
diff --git a/plugins/redfish/meson.build b/plugins/redfish/meson.build
index 1e35b6ce9..33ca1447f 100644
--- a/plugins/redfish/meson.build
+++ b/plugins/redfish/meson.build
@@ -35,7 +35,7 @@ plugin_builtin_redfish = static_library('fu_plugin_redfish',
 plugin_builtins += plugin_builtin_redfish
 
 install_data(['redfish.conf'],
-  install_dir: join_paths(sysconfdir, 'fwupd'),
+  install_dir: join_paths(sysconfdir_install, 'fwupd'),
   install_mode: 'rw-r-----',
 )

